src/Microsoft.Extensions.Logging.Abstractions/Internal/LogValuesFormatter.cs

```diff
@@ -20,6 +20,7 @@ public class LogValuesFormatter
 
         public string OriginalFormat { get; private set; }
         public List<string> ValueNames => _valueNames;
+        private const string NullValue = "(null)";
```

Comment:
While you're at it, you could move `OriginalFormat` and `ValueNames` after the ctor